bin/fetch-cards-pdf/main.go

```diff
@@ -6,135 +6,221 @@ import (
 	"io/ioutil"
 	"net/http"
 	"os"
+	"strconv"
 	"sync"
 
 	"github.com/PuerkitoBio/goquery"
+	"github.com/chibimi/cards/card/reference"
+	_ "github.com/go-sql-driver/mysql"
 	"github.com/gographics/imagick/imagick"
 	log "github.com/inconshreveable/log15"
+	"github.com/jmoiron/sqlx"
+	"github.com/pkg/errors"
 )
 
+type config struct {
+	ppURL         string
+	destDir       string
+	dsn           string
+	workers       int
+	secondaryOnly bool
+}
+
+type DownloadJob struct {
+	RefID string
+	Index int
```

Comment:
We may want to retrieve multiple cards for the same reference (thinking about colossals), so maybe `Indices []int` to avoid wasteful calls?

Reply:
We will see with https://github.com/chibimi/cards/issues/35